### PR TITLE
[Mempool] Trace.

### PR DESCRIPTION
**Pull request by grao1991**

### CODE REVIEW

File: mempool/src/core_mempool/transaction.rs [hunk 1]
```diff
@@ -34,6 +67,11 @@ impl MempoolTransaction {
         seqno: u64,
         insertion_time: SystemTime,
     ) -> Self {
+        let tracer = if (txn.sequence_number() & 255) < 100 {
```
Comment (msmouse) [inline mempool/src/core_mempool/transaction.rs:70]: 😂

Comment (msmouse) [reply mempool/src/core_mempool/transaction.rs:70]: Would like to see some comments around all the considerations.

**Review by msmouse — APPROVED** [review]

**Review by davidiw — APPROVED** [review]
per @msmouse comments.

**Comment by davidiw** [comment]
stale?

**Pull request closed without merging**